editor/animation_track_editor.cpp

```diff
@@ -2231,7 +2231,14 @@ void AnimationTrackEdit::draw_key(int p_index, float p_pixels_sec, int p_x, bool
 			if (i > 0) {
 				text += ", ";
 			}
-			text += String(args[i]);
+			String s = String(args[i]);
```

Comment:
To display the function parameters as code properly, just do it like this:
```C++
String s = args[i].get_construct_string();
```
and it should be enough.

Reply:
Ah, that's an interesting member I haven't noticed before. Going to try that one instead.